vendor/gradle-plugin/build.gradle

```diff
@@ -0,0 +1,36 @@
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.2'
+
+        // NOTE: Do not place your application dependencies here; they belong
+        // in the individual module build.gradle files
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
```

Comment:
shouldn't we add the google() repository as well?

Reply:
no need, because we decide what dependencies to use, and we don't have any dependencies that would use google()